README.md

```diff
@@ -49,7 +49,12 @@ conda activate kscale-sim-library
 make install-dev
 ```
 
-3. Install third-party dependencies:
+3. Configure environment variables:
```

Comment:
Please create a separate PR for the resources and update https://docs.kscale.dev/software/simulation/isaac as well